README.md

```diff
@@ -1,2 +1,49 @@
-# module-1-first-repository
- 
+# New Project as example!
+
+Hello!
+
+> Today I'm creating my first repository of Module 1 from Microverse.
+
+On this proyect I will work on the next concepts:
+
+- Git & Github Overflow
+- Linters for HTML, CSS and JS
+- Create Branch whit best practices
+- Best practices of commiting
+- Best practices for Pull Request
+
+Live Demo (Work on progress).
+
+## Working on my second day at Microverse!
+
+### This is Day 2 at Microverse, and I'm learning a lot of things in so a short time. I'm so excited to continue learning and breaking obstacles in the path. The morning session was so useful to know each partner of this one, and I think that I will enjoy solving some problems whit them in the future.
+
+There is the steps of the Day 2:
+
+- Morning session: Core study skills part 1
+
+- Test your knowledge: Microverse program requirements
+
+- Linters
+
+- Git & GitHub & GitHub Flow
+
+- Document your repo in a professional way
+
+**_Set up a "Hello Microverse" project_** **This is the actual step that I'm working on**
+
+- Code review flow
+
+- What does a code review really mean?
+
+- 1.2 HTML5 semantic tags, box model and developer tools
+
+- Images, Media and Backgrounds
+
+- Daily standup
+
+Contact me at:
+
+- GitHub: [BregornOriginal](https://github.com/BregornOriginal).
+
+![Esta es una imagen](https://camo.githubusercontent.com/38f113b96a368dfb7f634d2f2da97e7b8c748042d2a284b97c3fad048bb3ff55/68747470733a2f2f6d69726f2e6d656469756d2e636f6d2f6d61782f323733322f312a6d74736b3366515f4252656d466964686b656c3364412e706e67)
```

Comment:
Hi, @BregornOriginal!
As requested by the previous Code Reviewer, Please use this [template](https://github.com/microverseinc/readme-template) provided by Microverse 😄

Reply:
@zainsadaqat It's ok now?

Reply:
Hello @zainsadaqat , sorry if I'm bothering you, but this is my second request review. So I will be so glad if you can check the readme before I do another requested review. I don't want to repeat this for just a readme. Thank you!

Reply:
## Hey, @BregornOriginal! 😃 
Thanks for working on the requirements. You've successfully completed all the requirements for this project. You're good to go now. 🎉 
Please submit your PR for a review!